src/neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -265,6 +265,22 @@ public void Save()
             File.WriteAllText(path, wallet.ToString());
         }
 
+        public bool Backup()
```

Comment:
Can this method be moved to `Wallet.cs`?

Reply:
Or I think this is simply copying the file, there is no need to create a new method. The client can directly call `File.Copy()` to back up any file.

Reply:
ok, then we need `path` public.